agent/consul/operator_raft_endpoint_test.go

```diff
@@ -50,6 +50,13 @@ func TestOperator_RaftGetConfiguration(t *testing.T) {
 	if len(future.Configuration().Servers) != 1 {
 		t.Fatalf("bad: %v", future.Configuration().Servers)
 	}
+
+	serverIDLastIndexMap := make(map[raft.ServerID]uint64)
+
+	for _, serverState := range s1.autopilot.GetState().Servers {
+		serverIDLastIndexMap[serverState.Server.ID] = serverState.Stats.LastIndex
+	}
+
```

Comment:
This function only tests a single server `RaftConfiguration`. Since the feature being added is aimed at a cluster with more than one server, the entirety of the newly added code in this PR is not getting exercised.  If you can find prior art in the tests that use more than one `testServer(t)` (see L30), then it would be nice to take advantage. If not, don't consider this a blocker.

Reply:
Looked but could not find more than one `testServer(t)`